### ISSUE TITLE
ant\build.xml does not exist anymore

### ISSUE THREAD

**Issue by TheWlr9** [processing/processing-android#315]
Hello, I was installing the Android add-on to Processing and was super excited to start coding! I already have used Android Studio to code for Android devices already, so I selected the option for locating my sdk. I found my sdk and it recognized it as well. However, after I tried to compile and run my program, it gave me the following error: Cannot find C:\Users\Owner\AppData\Local\Android\sdk\tools\ant\build.xml .

So I decided to delve deep into the files to find the problem. The problem was that within the tools file, there was no ant file. (But that's a different topic that I see another user has already mentioned to you guys so I'll ignore it!) So I copied the ant file from my Android studio (because A.S. DOES come with ant), into the tools file. I tried running/compiling the program again.

The same error appeared. I checked, and I realized that the build.xml file does not exist in my ant file. That really stumped me because I couldn't just code the xml file! So I tried downloading just the ant file from a trustworthy mirror I found online. I looked inside its files and couldn't find the magical build.xml file I needed.

Finally, I looked inside one of the files in both ants and found that it has been renamed as "fetch.xml". I changed the name from fetch to build, and tried to compile my program, and now it works!

TL:DR: When users select the "locate sdk file" before installing the Android add-on for Processing, many (if not all) ant files now have the "build.xml" file renamed as "fetch.xml".

**Comment by codeanticode** [comment]
Hi, the latest version of the SDK tools have removed the ant build scripts and the android tool, thus breaking the mode. This is already reported in this other issue: https://github.com/processing/processing-android/issues/307
